src/widget/wwidgetgroup.cpp

```diff
@@ -152,6 +147,15 @@ void WWidgetGroup::setup(const QDomNode& node, const SkinContext& context) {
 
     if (pLayout) {
         setLayout(pLayout);
+
+        if (layout == "stacked") {
```

Comment:
Using this opportunity to introduce a constant would help. Even if the remaining code still uses string literals.

Reply:
In this case you may also/instead cache the result of the first string comparison in a boolean.

Reply:
Probably not worth it the extra code and maintenance burden.

Reply:
I went for the added boolean, I think it's quite clear like this.